deepchem/feat/base_classes.py

```diff
@@ -58,6 +60,8 @@ def featurize(self, datapoints: Iterable[Any],
   def __call__(self, datapoints: Iterable[Any]):
     """Calculate features for datapoints.
 
+    `**kwargs` will get passed directly to `Featurizer.featurize`
```

Comment:
I might be off here but this reads a little cryptic to me since Featurizer.featurize then says **kwargs will get passed to _featurize, which also does not seem to handle **kwargs.

Reply:
yup this is an old comment from when we passed **kwargs to _featurize - will delete this